train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -41,9 +44,17 @@ private void enforceSpeedLimit() {
 		}
 	}
 
+
 	@Override
 	public void setJoystickPosition(int joystickPosition) {
-		this.step = joystickPosition;		
+		this.step = joystickPosition;
+		Timer timer = new Timer();
+		timer.schedule(new TimerTask(){
+			@Override
+			public void run() {
+				followSpeed();
+			}
+		}, 2000);
```

Comment:
Here: this value could use some comments.

Reply:
It is a clear context which doesn't need any explanation.

Reply:
kk